### PR TITLE
Feature/sportec tracking

### PR DESCRIPTION
**Pull request by koenvo**
- [x] Deprecated `sportec.load` and added `sportec.load_event`
- [x] Add deserializer for Sportec Tracking data
- [x] Update docs

### CODE REVIEW

File: kloppy/_providers/sportec.py [hunk 1]
```diff
@@ -36,5 +37,20 @@ def load(
         meta_data
     ) as meta_data_fp:
         return serializer.deserialize(
-            SportecInputs(event_data=event_data_fp, meta_data=meta_data_fp)
+            SportecEvenDataInputs(
```
Comment (MKlaasman) [inline kloppy/_providers/sportec.py:44]: Should be: SportecEventDataInputs

File: kloppy/_providers/sportec.py [hunk 2]
```diff
@@ -3,13 +3,14 @@
 from kloppy.config import get_config
 from kloppy.domain import EventDataset, EventFactory
 from kloppy.infra.serializers.event.sportec import (
-    SportecEventDeserializer,
-    SportecInputs,
+    SportecEventDataDeserializer,
+    SportecEvenDataInputs,
```
Comment (MKlaasman) [inline kloppy/_providers/sportec.py:7]: Should be: SportecEventDataInputs

File: kloppy/tests/test_sportec.py [hunk 3]
```diff
@@ -69,9 +77,48 @@ def test_correct_deserialization(self, event_data: Path, meta_data: Path):
...
+
+        assert dataset.frames[1].ball_owning_team == home_team
+        assert dataset.frames[1].ball_state == BallState.ALIVE
+
+        player_lilian = away_team.get_player_by_id("DFL-OBJ-002G3I")
+        player_data = dataset.frames[0].players_data[player_lilian]
+
+        assert player_data.coordinates == Point(x=0.35, y=-25.26)
+
+        # We don't load distance right now as it doesn't
```
Comment (MKlaasman) [inline kloppy/tests/test_sportec.py:133]: There is also an `Acceleration` attribute in the Sportec data, is there a possibility to add this to the dataset?

File: kloppy/infra/serializers/event/sportec/deserializer.py [hunk 4]
```diff
@@ -74,6 +66,106 @@ def _team_from_xml_elm(team_elm) -> Team:
...
+    teams: List[Team]
+    periods: List[Period]
+    x_max: float
+    y_max: float
+    fps: int
+
+
+def _sportec_metadata_from_xml_elm(match_root) -> SportecMetadata:
+    """
+    Load metadata from Sportec XML element. This part is shared between event- and tracking data.
```
Comment (MKlaasman) [inline kloppy/infra/serializers/event/sportec/deserializer.py:83]: Nice!

**Review by MKlaasman — COMMENTED** [review]
Thanks for this PR Koen! 

Overall this looks as I expected, I think there is still some value in adding `Acceleration` to the dataset.

File: kloppy/tests/test_sportec.py (hunk 3 above)
Comment (koenvo) [reply kloppy/tests/test_sportec.py:133]: Jup. Would like to implement this in a different ticket to first check other providers too.

File: kloppy/infra/serializers/tracking/sportec/deserializer.py [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+                                            x=float(raw_player_data["X"]),
+                                            y=float(raw_player_data["Y"]),
+                                        ),
+                                        speed=float(raw_player_data["S"]),
+                                    )
+                                    for player_id, raw_player_data in frame_data.items()
+                                    if player_id != "ball"
+                                },
+                                other_data={},
+                                ball_coordinates=Point3D(
```
Comment (koenvo) [inline kloppy/infra/serializers/tracking/sportec/deserializer.py:187]: Add ball_speed

**Comment by UnravelSports** [comment]
I've ran it and it seems to be working correctly after you added ball speed and the updated coordinate systems!

File: kloppy/domain/services/transformers/dataset.py [hunk 6]
```diff
@@ -431,3 +435,53 @@ def transform_dataset(
...
+
+    def build(
+        self,
+        length: float,
+        width: float,
+        provider: Provider,
+        dataset_type: DatasetType,
+    ):
+        from_coordinate_system = build_coordinate_system(
+            provider, length=length, width=width, dataset_type=dataset_type
```
Comment (JanVanHaaren) [inline kloppy/domain/services/transformers/dataset.py:474]: Perhaps use the same formatting as for the `to_coordinate_system`? The similarties and differences between the two calls to `build_coordinate_system` would become more apparent.

File: kloppy/infra/serializers/event/sportec/deserializer.py [hunk 7]
```diff
@@ -74,6 +66,106 @@ def _team_from_xml_elm(team_elm) -> Team:
...
+
+    # The periods can be rebuild from event data. Therefore, the periods attribute
+    # from the metadata can be ignored. It is required for tracking data.
+    other_game_information = (
+        match_root.MatchInformation.OtherGameInformation.attrib
+    )
+    periods = [
+        Period(
+            id=1,
+            start_timestamp=10_000 / SPORTEC_FPS,
```
Comment (JanVanHaaren) [inline kloppy/infra/serializers/event/sportec/deserializer.py:126]: It is not immediately clear to me where the `10_000`, `100_000`, `200_000` and `250_000` constant values come from. I suspect it is a Sportec convention. How about moving these constant values to a variable with a more descriptive name? Or adding a quick comment explaining the convention?

**Review by JanVanHaaren — APPROVED** [review]
The pull request looks good to me! I added two minor suggestions to improve the readability of the code.

**Pull request merged**